services/web-app/pages/about-carbon/how-carbon-works.mdx

```diff
@@ -67,25 +67,26 @@ as color tokens, type tokens, spacing tokens, the grid, and more.
 **Guidelines** provide direction on critical topics such as accessibility and content and ensure
 consistent user experiences that meet IBM's standards.
 
-**Assets** include re–usable resources such as components, patterns, functions, and templates. Each
-asset has code, designs, and usage guidance.
+**Assets** are re–usable resources such as components, patterns, functions, and templates. Assets
+often have usage and accessibililty guidance, as well as styling specifications and code.
 
-**Design kits** are libraries from Sketch and Figma—and other tools such as Adobe XD and Invision.
-They contain components and elements as well as wireframes and guidance to aid designers.
+**Design kits** contain elements, guidelines, user interface components and patterns, and wireframes
+for various design tools as the design counterpart to coded assets.
 
-**Libraries** are 1:1 with code packages. All coded assets belong to a library and have a
-maintainer. Design kits with compatible code also live in libraries.
+**Libraries** are the means to contribute, install, and use one or many assets. All coded assets
+belong to a library. Most all libraries are published as code packages. Libraries have compatible
+design kits.
 
-## Carbon maintainers
+## Maintainers
 
-Carbon core and local systems have `maintainers` that administer governance over content and
-resource visibility. Each resource cites a maintaining team if an IBM team primarily maintains the
-resource. If an individual or informal group is responsible for a resource, it will be listed as
-community maintained.
+Carbon systems have maintainers that administer governance over content and resource visibility.
+Design kits and libraries cite a maintaining team if an IBM team primarily maintains the resource.
```

Comment:
Maybe say "Libraries and design kits" with libraries first otherwise it could read like "Design kits and (design implied) libraries..."